### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by srarcbrsent**

### CODE REVIEW

**Comment by srarcbrsent** [comment]
@star-seeker 👻👻👻👻👻

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # star_seeker
+
 My first github repository
 
 For a long time, I have been alive just not for myself.
 
 Life is like a sword
+
+👻👻👻👻👻👻👻👻👻👻
```
Comment (srarcbrsent) [inline README.md:9]: 👻

Comment (srarcbrsent) [reply README.md:9]: #4

**Comment by star-seeker** [comment]
😱😱😱😱😱

**Comment by srarcbrsent** [comment]
居然给close了没有accept

**Comment by star-seeker** [comment]
~~

**Pull request merged**